### PR TITLE
Update usage-scenario-shard-merge.md (#224)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #224 to release-1.0

---

### What is changed, added or deleted? (Required)

add scenario description

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/209
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: en/usage-scenario-shard-merge.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
 ---
 title: Data Migration Shard Merge Scenario
+<<<<<<< HEAD
```
Comment (yikeke) [inline en/usage-scenario-shard-merge.md:3]: Please resolve conflicts. @ireneontheway

Comment (yikeke) [inline en/usage-scenario-shard-merge.md:3]: Please resolve conflicts. @ireneontheway

Comment (ireneontheway) [reply en/usage-scenario-shard-merge.md]: Done

File: en/usage-scenario-shard-merge.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 ---
 title: Data Migration Shard Merge Scenario
-summary: Learn how to use Data Migration to replicate data in the shard merge scenario.
-aliases: ['/docs/tidb-data-migration/stable/usage-scenario-shard-merge/','/docs/tidb-data-migration/v1.0/usage-scenario-shard-merge/','/docs/dev/reference/tools/data-migration/usage-scenarios/shard-merge','/docs/v3.1/reference/tools/data-migration/usage-scenarios/shard-merge','/docs/v3.0/reference/tools/data-migration/usage-scenarios/shard-merge','/docs/v2.1/reference/tools/data-migration/usage-scenarios/shard-merge','/docs/stable/reference/tools/data-migration/usage-scenarios/shard-merge/']
+summary: Learn how to use Data Migration to migrate data in the shard merge scenario.
+aliases: ['/docs/tidb-data-migration/dev/usage-scenario-shard-merge/']
```
Comment (yikeke) [inline en/usage-scenario-shard-merge.md:4]: The original PR https://github.com/pingcap/docs-dm/pull/224/ does not change the aliases, so do not change aliases in the cherry-pick PR. @ireneontheway

Comment (ireneontheway) [reply en/usage-scenario-shard-merge.md]: Done

**Pull request merged**